lib/_tls_wrap.js

```diff
@@ -1113,6 +1113,11 @@ Server.prototype.setSecureContext = function(options) {
   else
     this.crl = undefined;
 
+  if (options.sigalgs)
```

Comment:
maps all the "falsy" values to undefined, better to be explicit

Reply:
Like `options.sigalgs === undefined`? Looks like the same check is using for all the other options, including the `ciphers` that has pretty much the same expected format.